### PR TITLE
Add documenation for new actionlint analyzer

### PR DESCRIPTION
**Pull request by radomirbosak**
Related PRs:
* runners: https://github.com/sider/runners/pull/2650

### CODE REVIEW

**Comment by radomirbosak** [comment]
Prettier check fails,
```shell
yarn run v1.22.11
$ npm run prettier -- --check
npm WARN lifecycle The node binary used for scripts is /tmp/yarn--1631235095251-0.8794903907620224/node but npm is using /opt/hostedtoolcache/node/14.17.6/x64/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> @ prettier /home/runner/work/sider-docs/sider-docs
> prettier --ignore-path=.gitignore . "--check"

Checking formatting...
[warn] docs/tools/others/actionlint.md
[warn] Code style issues found in the above file(s). Forgot to run Prettier?
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ prettier: `prettier --ignore-path=.gitignore . "--check"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ prettier script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-09-10T00_51_38_567Z-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```
but I cannot reproduce this on my local machine
```shell
$ npm run prettier --ignore-path=.gitignore .
```
so I don't know what is wrong.

**Review by dev-yakuza — APPROVED** [review]
LGTM 👍

File: docs/tools/others/actionlint.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      - .github/workflows/one_worflow.yml
+      - .github/workflows/another_worflow.yml
+    ignore:
+      - "job .* needs job"
+      - "unknown Webhook event"
+```
+
+| Name                | Type                 | Default |
+| ------------------- | -------------------- | ------- |
+| [`target`](#target) | `string`, `string[]` | -       |
```
Comment (dev-yakuza) [inline docs/tools/others/actionlint.md:42]:
```suggestion
| [`root_dir`](../../getting-started/custom-configuration.md#linteranalyzer_idroot_dir) | `string`             | -       |
| [`target`](#target) | `string`, `string[]` | -       |
```

I think we need to add `root_dir` option here like other documents. 🤔 

https://help.sider.review/tools/ruby/rubocop#configuration

Comment (radomirbosak) [reply docs/tools/others/actionlint.md:43]: Changed. Thank you.

File: docs/tools/others/actionlint.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## Configuration
+
+You can customize the analysis via `sider.yml`:
+
+```yaml
+linter:
+  actionlint:
+    target:
+      - .github/workflows/one_worflow.yml
+      - .github/workflows/another_worflow.yml
```
Comment (suetsuguk) [inline docs/tools/others/actionlint.md:34]: typo.

```suggestion
      - .github/workflows/one_workflow.yml
      - .github/workflows/another_workflow.yml
```

Comment (radomirbosak) [reply docs/tools/others/actionlint.md]: Fixed. Thank you.

**Review by ybiquitous — APPROVED** [review]
Looks good! This PR should be merged when actionlint will be available on <sider.review>.

File: docs/tools/others/actionlint.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+| Name                                                                                  | Type                 | Default |
+| ------------------------------------------------------------------------------------- | -------------------- | ------- |
+| [`root_dir`](../../getting-started/custom-configuration.md#linteranalyzer_idroot_dir) | `string`             | -       |
+| [`target`](#target)                                                                   | `string`, `string[]` | -       |
+| [`ignore`](#ignore)                                                                   | `string`, `string[]` | -       |
+
+### `target`
+
+This option allows you to specify files to analyze via actionlint. Glob format is _not_ available.
+If target is not specified, by default, all yaml files in the `.github/workflow/` directory of the repository root are analyzed.
```
Comment (suetsuguk) [inline docs/tools/others/actionlint.md:49]: typo.

```suggestion
If target is not specified, by default, all yaml files in the `.github/workflows/` directory of the repository root are analyzed.
```

Comment (radomirbosak) [reply docs/tools/others/actionlint.md]: Nice catch! Thank you.

**Review by suetsuguk — APPROVED** [review]
LGTM. Thank you.

**Pull request merged**